### DISCUSSION TITLE
blog/jellyfin-arr-stack-selfhosted-media-streaming-in-my-homelab/

### DISCUSSION THREAD

**Discussion by giscus** [AkashRajpurohit/AkashRajpurohit#38] [Comments]
# blog/jellyfin-arr-stack-selfhosted-media-streaming-in-my-homelab/

Since ages, I have been collecting lots of movies, TV shows, and music. Ever since I got into self hosting, I have been looking for a way to stream my media collection to my devices. Jellyfin is the perfect solution for this.

https://akashrajpurohit.com/blog/jellyfin-arr-stack-selfhosted-media-streaming-in-my-homelab/

**Comment by benjaminstart** [comment]
Looking forward to your post on how to run Jellyfin on an Orange Pi 5 Plus. How good is the 4k transcoding (say compared to QSV on an N100)?

**Reply by AkashRajpurohit** [reply]
I don't have personal experience with QSV on N100, but I'll share stats from my experience with hw acceleration on Orange Pi 5 Plus.

The blog is in works and hopefully will be coming out soon.

**Reply by AkashRajpurohit** [reply]
@benjaminstart Quick update, here's the blog for Jellyfin on orange pi 5 with HW transcoding: https://akashrajpurohit.com/blog/setup-jellyfin-with-hardware-acceleration-on-orange-pi-5-rockchip-rk3558/
